### PR TITLE
add bot view + dialogflow sdk

### PR DESCRIPTION
**Pull request by elainedb**

### CODE REVIEW

File: app/build.gradle [hunk 1]
```diff
@@ -78,8 +78,19 @@ dependencies {
     // Used for tags
     implementation 'org.apmem.tools:layouts:1.10@aar'
 
-    testImplementation 'junit:junit:4.12'
-    androidTestImplementation('com.android.support.test.espresso:espresso-core:2.2.2', {
+    // Butterknife
+    compile 'com.jakewharton:butterknife:8.5.1'
+    annotationProcessor 'com.jakewharton:butterknife-compiler:8.5.1'
```
Comment (benju69) [inline app/build.gradle]: Pas de Butterknife pls :s databinding ou des findview

File: app/google-services.json [hunk 2]
```diff
@@ -0,0 +1,42 @@
+{
```
Comment (benju69) [inline app/google-services.json]: Je sais pas si on commit ça sur le repo

File: app/build.gradle (hunk 1 above)
Comment (martinbonnin) [reply app/build.gradle]: on peut négocier des synthetic properties ? ;)

**Comment by elainedb** [comment]
butterknife removed
google-services.json removed

File: app/src/main/res/values/strings.xml [hunk 3]
```diff
@@ -94,4 +95,15 @@
     <string name="location_sponsor">Location</string>
     <string name="specialthanks_sponsor">Special Thanks</string>
 
+    <!--Bot-->
+    <string name="bot_hello">Hi! I am MakerDroid. You can ask me questions about the Android Makers agenda, like what are the current sessions. How can I help you?</string>
```
Comment (benju69) [inline app/src/main/res/values/strings.xml]: more questions examples : next sessions, in english ?

File: app/build.gradle [hunk 4]
```diff
@@ -87,6 +87,11 @@ dependencies {
 
     // For Video preview
     implementation files('libs/YouTubeAndroidPlayerApi.jar')
+
+    // Dialogflow SDK dependencies
+    compile 'ai.api:sdk:2.0.7@aar'
+    compile "com.android.support:appcompat-v7:$play_services_version"
```
Comment (benju69) [inline app/build.gradle]: useless

File: app/build.gradle [hunk 5]
```diff
@@ -87,6 +87,11 @@ dependencies {
...
+    compile 'ai.api:libai:1.6.12'
```
Comment (benju69) [inline app/build.gradle]: Now we use "implementation"

**Comment by benju69** [comment]
impossible to build, any idea on this errors ?

Configuration 'compile' is obsolete and has been replaced with 'implementation' and 'api'. |  
-- | --
org.slf4j.LoggerFactory: can't find referenced class org.slf4j.impl.StaticLoggerBinder |  
org.slf4j.MDC: can't find referenced class org.slf4j.impl.StaticMDCBinder |  
org.slf4j.MarkerFactory: can't find referenced class org.slf4j.impl.StaticMarkerBinder |  
there were 13 unresolved references to classes or interfaces. |  
Exception while processing task java.io.IOException: Please correct the above warnings first. |

**Comment by benju69** [comment]
I think there is some proguard rules to add

**Comment by benju69** [comment]
Proguard fixed after merge

**Pull request merged**